src/main/java/com/r307/arbitrader/service/model/ExchangeFee.java

```diff
@@ -0,0 +1,38 @@
+package com.r307.arbitrader.service.model;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+public class ExchangeFee {
+    private BigDecimal shortFee;
+    private BigDecimal longFee;
+
+    public ExchangeFee(BigDecimal shortFee, BigDecimal longFee) {
```

Comment:
I hate to nitpick but everywhere else in the code I've been careful to always do long, then short. Since both these arguments are the same type it's going to be an easy mistake to make if everything else is the other way around.

Reply:
Good point :+1: